src/interactions/chatInput/staffapp.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  ActionRowBuilder,
+  ApplicationCommandOptionType,
+  CacheType,
+  CommandInteraction,
+  ModalActionRowComponentBuilder,
+  ModalBuilder,
+  TextInputBuilder,
+  TextInputStyle,
+} from "discord.js";
+import {
+  CustomInteractionReplyOptions,
+  InteractionCommand,
+} from "../../classes/CustomInteraction";
+
+import { ApplicationCommandType } from "discord-api-types/v10";
+import { CustomClient } from "lib/client";
+
+export default class SlashCommand extends InteractionCommand {
+  /**
+   *
+   */
+  constructor(client: CustomClient) {
+    super(client, {
+      type: ApplicationCommandType.ChatInput,
+      name: "staff",
+      options: [
+        {
+          type: ApplicationCommandOptionType.Subcommand,
+          name: "apply",
+          description: "apply for staff",
+        },
+      ],
+      description: "apply for staff",
+    });
+  }
+
+  async execute(
+    interaction: CommandInteraction<CacheType>
+  ): Promise<CustomInteractionReplyOptions> {
+    const modal = new ModalBuilder().setCustomId("s").setTitle("Staff App");
+
+    // Add components to modal
+
+    // Create the text input components
+    const goodPersonInput = new TextInputBuilder()
+      .setCustomId("goodPersonInput")
+      .setLabel("Are you a good person?")
+      .setRequired(true)
+      .setStyle(TextInputStyle.Paragraph);
```

Comment:
Read previous comment.

Also take a look at the previous applications for which fields we want people to input.